enhancements/node-healthcheck-operator.md

```diff
@@ -0,0 +1,449 @@
+Node Health Check Enhancement 
+
+---
+title: node-healthcheck-controller
+authors:
+  - "@rgolangh"
+reviewers:
+  - "@abeekhof"
+  - "@n1r1"
+  - "@mshitrit"
+  - "@slintes"
+  - "@dhellmann"
+approvers:
+  - "@abeekhof"
+  - "@n1r1"
+---
+
+# Node health checking
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs
+
+
+## Summary
+
+
+This enhancement proposes the creation of a new OLM-delivered operator
+that supports NodeCondition-based health checking of UPI worker nodes.
+
+The new operator utilises the same External Remediation API as Machine 
+Healthcheck to trigger recovery of affected nodes and workloads.
+
+## Motivation
+
+Enable clusters that do not have a functioning Machine API to detect and respond to node-level failures in order to recover workloads , and/or restore cluster capacity.
+
+The majority of OpenShift clusters are UPI based and do not have Machine objects
+and/or a functioning Machine API - which is a prerequisite for using the Machine 
+HealthCheck controller.
+
+There are a variety of reasons, even on cloud, that customers might want or need
+to pursue a UPI strategy *for deployment*, but that doesn't need to prevent them
```

Comment:
It'd be nice to mention a few to make it clear why moving to mapi automated machine management is not an option.